build.sbt

```diff
@@ -6,7 +6,7 @@ organization in ThisBuild := "org.julienrf"
 
 scalaVersion in ThisBuild := "2.13.0"
 
-crossScalaVersions := Seq("2.10.6", "2.11.12", "2.12.8", scalaVersion.value)
+crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.8", scalaVersion.value)
```

Comment:
Interesting. The tests fail on 2.11.12? Is that a known issue?

Reply:
I didn't really bother to go down that rabbit hole. Perhaps a `clean` would have been sufficient.